mock/py/mockbuild/plugins/buildroot_lock.py

```diff
@@ -0,0 +1,111 @@
+"""
+Produce a lockfile for the prepared buildroot by Mock.  Once available, we
+should use the DNF built-in command from DNF5:
+https://github.com/rpm-software-management/dnf5/issues/833
+"""
```

Comment:
I'm explicitly linking this PR to https://github.com/rpm-software-management/dnf5/issues/833